core/src/main/java/io/grpc/internal/InternalSubchannel.java

```diff
@@ -772,6 +772,9 @@ private String printShortStatus(Status status) {
     if (status.getDescription() != null) {
       buffer.append("(").append(status.getDescription()).append(")");
     }
+    if (status.getCause() != null) {
+      buffer.append("(").append(status.getCause()).append(")");
```

Comment:
It may be better to differentiate between the description and the cause. Maybe use brackets to surround it.